release-notes.md

```diff
@@ -44,6 +44,13 @@ This document contains release notes per zib, indicating differences with their
 * `PracitionRole.speciality` is not sliced anymore but has a combined ValueSet.
 * Explained cardinality mismatch of Practitioner.name on that element.
 
+## zib-InstructionsForUse
+* Removed required GstdTabel902 bindings on Quantity datatypes for Dose and MaximumDose. Added pattern-GstdQuantity profile in nl-core profile that contains a PQ-translation extension that can hold the GstdTabel902 binding.
+* Moved mapping of Description from `Dosage.text` to an extension that is placed in the host resource. This was already fixed with MP9.1, however, it was forgotten in the zib2017 stable2.x branch.
```

Comment:
I can think of a reason, but why is this done? Shouldn't this be documented in the profile as well?

Reply:
It is documented in the profile on the root. Could you check if the wording is good enough?

Reply:
Yeah, about that ... I removed the mapping and the concept description from this extension. There's no reason not to make this a generic extension and in fact, its name suggests that it is.

Reply:
It is a generic extension, but used/pertaining to only one zib and one concept. Wouldn't it be easier to keep the mapping in the extension? I don't quite get the difference with ext-InstructionsForUse.RepeatPeriodCyclicalSchedule.

Reply:
Discussed!